probe_scraper/scraper.py

```diff
@@ -0,0 +1,192 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import re
+import os
+import json
+import tempfile
+import requests
+import requests_cache
+
+from collections import OrderedDict
+
+requests_cache.install_cache('probe_scraper_cache')
+
+
+REGISTRY_FILES = {
+    'histograms': [
+        'toolkit/components/telemetry/Histograms.json',
+        'dom/base/UseCounters.conf',
+        'dom/base/nsDeprecatedOperationList.h',
+    ],
+    'scalars': [
+        'toolkit/components/telemetry/Scalars.yaml',
+    ],
+    'events': [
+        'toolkit/components/telemetry/Events.yaml',
+    ],
+}
+
+CHANNELS = {
+    # 'nightly': 'https://hg.mozilla.org/mozilla-central/',
+    'aurora': {
+        'base_uri': 'https://hg.mozilla.org/releases/mozilla-aurora/',
+        'tag_regex': '^FIREFOX_AURORA_[0-9]+_BASE$'
+    },
+    'beta': {
+        'base_uri': 'https://hg.mozilla.org/releases/mozilla-beta/',
+        'tag_regex': '^FIREFOX_BETA_[0-9]+_BASE$',
+    },
+    'release': {
+        'base_uri': 'https://hg.mozilla.org/releases/mozilla-release/',
+        'tag_regex': '^FIREFOX_[0-9]+_0_RELEASE$',
+    },
+}
+
+MIN_FIREFOX_VERSION = 30
+ERROR_CACHE_FILENAME = 'probe_scraper_errors_cache.json'
+
+
+def load_tags(channel):
+    uri = CHANNELS[channel]['base_uri'] + "json-tags"
+    r = requests.get(uri)
+    if r.status_code != requests.codes.ok:
+        raise Exception("Request returned status " + str(r.status_code) + " for " + uri)
+
+    ctype = r.headers['content-type']
```

Comment:
ctype is the name of a module in the python standard library. Please change it to `content_type` which is also more meaningful.